settings.gradle

```diff
@@ -232,3 +232,6 @@ include "code/stack/View"
 project(":code/stack/View").name = "stack-View"
 include "code/stack/ViewModel"
 project(":code/stack/ViewModel").name = "stack-ViewModel"
+// Prime Number
+include "code/prime-number/Model"
+project(":code/prime-number/Model").name = "prime-number-Model"
```

Comment:
Please, put these lines instead your name in settings.gradle
And make sure, that your PR doesn't call merge conflicts